### DISCUSSION TITLE
Android client & DB

### DISCUSSION THREAD

**Discussion by TomBenDor** [TomBenDor/AP2#228] [Announcements]
## What's Changed
* Milestone 1: Web Client by @TomBenDor in https://github.com/TomBenDor/AP2/pull/121
* Mention React libraries used in README.md by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/139
* Add closed issues to Development link by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/140
* Init ASP.NET by @TomBenDor in https://github.com/TomBenDor/AP2/pull/141
* Create dotnet.yml by @TomBenDor in https://github.com/TomBenDor/AP2/pull/142
* Renaming server to reviews-website by @TomBenDor in https://github.com/TomBenDor/AP2/pull/152
* Create Class library and Web API by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/153
* Create Sign up method in Web API by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/158
* Communicate with remote servers by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/160
* Allow all origins to interact with the API by @TomBenDor in https://github.com/TomBenDor/AP2/pull/163
* Add JWT authentication by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/161
* Add hard coded users to database by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/166
* Return users from Web-API successfully by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/165
* Embed React by @TomBenDor in https://github.com/TomBenDor/AP2/pull/167
* Adding contact via API by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/170
* Support individual contact names by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/172
* Change server port by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/176
* Reviews Page by @TomBenDor in https://github.com/TomBenDor/AP2/pull/168
* Connect reviews site to React app by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/175
* Cleanup by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/180
* Removed some bears by @TomBenDor in https://github.com/TomBenDor/AP2/pull/183
* Milestone 2 Readme by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/182
* Milestone 2 - API + Reviews by @TomBenDor in https://github.com/TomBenDor/AP2/pull/181
* Init Android by @TomBenDor in https://github.com/TomBenDor/AP2/pull/193
* Downgrade Web-API to http by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/189
* Create basic Android screens by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/195
* Create Repository and Contacts Recycler View by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/201
* Android Authentication by API by @TomBenDor in https://github.com/TomBenDor/AP2/pull/202
* Reload contacts on refresh swipe by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/207
* Messages RecyclerView by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/205
* Add dark theme by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/208
* Fix dark mode settings switch by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/212
* Design custom toolbars by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/215
* Support profile picture uploading by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/216
* Shorten last message when needed to display by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/221
* Move SignalR to server by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/224
* Embed API by @TomBenDor in https://github.com/TomBenDor/AP2/pull/217
* Edit Server address in settings  by @Tom-stack3 in https://github.com/TomBenDor/AP2/pull/223
* Connect Firebase by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/203
* Fix firebase message by @OmerShoulstein in https://github.com/TomBenDor/AP2/pull/226

**Full Changelog**: https://github.com/TomBenDor/AP2/compare/web-client...android

<hr /><em>This discussion was created from the release <a href='https://github.com/TomBenDor/AP2/releases/tag/android'>Android client & DB</a>.</em>

**Comment by Tom-stack3** [comment]
Hurray! :partying_face:
